two_factor/templates/two_factor/twilio/sms_message.html

```diff
@@ -0,0 +1,5 @@
+{% load i18n %}
+{% blocktrans %}
```

Comment:
`{% blocktrans trimmed %}` is better for translators (it removes unnecessary whitespace)